keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -0,0 +1,730 @@
+  - 99,9% of /health requests per day finish with 200 code
+
+* **Are there any missing metrics that would be useful to have to improve observability 
+of this feature?**
+  Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+  implementation difficulties, etc.).
+
+### Dependencies
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **Does this feature depend on any specific services running in the cluster?**
+  Think about both cluster-level services (e.g. metrics-server) as well
+  as node-level agents (e.g. specific version of CRI). Focus on external or
+  optional services that are needed. For example, if this feature depends on
+  a cloud provider API, or upon an external software-defined storage or network
+  control plane.
+
+  For each of these, fill in the following—thinking about running existing user workloads
+  and creating new ones, as well as about cluster-level services (e.g. DNS):
+  - [Dependency name]
+    - Usage description:
+      - Impact of its outage on the feature:
+      - Impact of its degraded performance or high-error rates on the feature:
+
+
+### Scalability
+
+_For alpha, this section is encouraged: reviewers should consider these questions
+and attempt to answer them._
+
+_For beta, this section is required: reviewers must answer these questions._
+
+_For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field._
+
+* **Will enabling / using this feature result in any new API calls?**
+  It should not.
+
+* **Will enabling / using this feature result in introducing new API types?**
+  Describe them, providing:
+  - There will be new CRI API types, described above. These are to be agreed upon by Kubelet and the CRI implementation.
+
+* **Will enabling / using this feature result in any new calls to the cloud 
+provider?**
+  - No.
+* **Will enabling / using this feature result in increasing size or count of 
+the existing API objects?**
+  Describe them, providing:
+  - Yes, in the CRI API.
+ 
+* **Will enabling / using this feature result in increasing time taken by any 
+operations covered by [existing SLIs/SLOs]?**
+  Think about adding additional work or introducing new steps in between
+  (e.g. need to do X to start a container), etc. Please describe the details.
+  - The amount of work should not be too different, depending on the CRI implementation's implementation of the new endpoints.
+  - It should not start pod startup, but possibly CPU usage on the node. This should be mitigated by Beta.
+* **Will enabling / using this feature result in non-negligible increase of 
+resource usage (CPU, RAM, disk, IO, ...) in any components?**
+  - Possibly, depending on the CRI implementation. This issues should be resolved to graduate to Beta.
+  - It is unlikely they will be as bad as the CRI implementations duplicating work with cAdvisor as they are now.
```

Comment:
If anything, this KEP should reduce resource utilization. We might want to call that out, and also include it as something to measure/evaluate for beta.

Reply:
I've updated the description a bit to better reflect what is the likely difference in performance